rfs/src/pack.rs

```diff
@@ -135,6 +139,14 @@ async fn pack_one<S: Store>(
         let meta = child.metadata().await?;
         let child_path = path.join(&name);
 
+        if sender.is_some() {
+            sender
+                .clone()
+                .unwrap()
+                .send(1)
+                .context("failed to send progress")?;
+        }
+
```

Comment:
```suggestion
      if let Some(sender) = sender {
          sender.send(1).context("failed to send progress")?;
      }
```
- There is no need to copy the sender (unecessary) 
- checking `is_some()` and then doing unwrap is very **un**rust